frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/mention/mention_block.dart

```diff
@@ -91,6 +93,19 @@ class MentionBlock extends StatelessWidget {
           textStyle: textStyle,
           index: index,
         );
+      case MentionType.childPage:
+        final String? pageId = mention[MentionBlockKeys.pageId] as String?;
+        if (pageId == null) {
+          return const SizedBox.shrink();
+        }
+        return MentionSubPageBlock(
+          key: ValueKey(pageId),
+          editorState: editorState,
+          pageId: pageId,
+          node: node,
+          textStyle: textStyle,
+          index: index,
```

Comment:
Getting a `cannot find BlocProvider<DocumentBloc> above this BlocConsumer<MentionSubPageBloc, MentionSubPageState>` when dragging the block